db/db.js

```diff
@@ -0,0 +1,47 @@
+const config = require('config');
+const _ = require('lodash');
+const oracledb = require('oracledb');
+const StaffFeePrivilegeSerializer = require('../serializers/staff-fee-privilege');
+
+const dbConfig = config.get('database');
+oracledb.outFormat = oracledb.OBJECT;
+
+process.on('SIGINT', () => process.exit());
+
+// Sanitize raw data from database
+const sanitize = (row) => {
+  row.CAMPUS = row.CAMPUS.trim();
+  row.CURRENT_ENROLLED = row.CURRENT_ENROLLED === 'Y';
+  row.CURRENT_REGISTERED = row.CURRENT_REGISTERED === 'Y';
+  return row;
+};
+
+const getStaffFeePrivilegesBy = (filter, query) =>
+  new Promise(async (resolve, reject) => {
+    let connection;
+    try {
+      oracledb.createPool(dbConfig, async (err, pool) => {
```

Comment:
As we discussed, we'll want to have the connection pool created when the application starts up and keep the pool open while the application is running.

Reply:
Good catch. Fixed.